src/dynamics/solver/contact/mod.rs

```diff
@@ -91,6 +91,7 @@ impl ContactConstraint {
         collider_entity2: Entity,
         collider_transform1: Option<ColliderTransform>,
         collider_transform2: Option<ColliderTransform>,
+        collision_margin: Scalar,
```

Comment:
```suggestion
        collision_margin: CollisionMargin,
```
if we already have the newtypes, we should also be consistently using them imo